libs/liquidity/src/lib/liquidity-table.spec.tsx

```diff
@@ -65,7 +86,28 @@ describe('LiquidityTable', () => {
       'Created',
       'Updated',
     ];
+    // 5002-LIQP-001
     expect(headers).toHaveLength(expectedHeaders.length);
     expect(headerTexts).toEqual(expectedHeaders);
   });
+
+  it('should be able to sort', async () => {
+    await act(async () => {
+      render(
+        <LiquidityTable rowData={multiRowData} stakeToCcyVolume={'0.3'} />
+      );
+    });
+    const headers = await screen.findAllByRole('columnheader');
+    await userEvent.click(headers[7]);
```

Comment:
I would like to see if there is a better way to locate this stuff than by index. And given we are in monorepo now, if it is hard to locate we should be encouraging dev to add data-testids that make this stuff easier, or adding them ourselves, this way of locating is pretty brittle and will fail the second we get a new header at any earlier index.

Reply:
I've updated this to use col-id